packages/dx-grid-core/src/types/table.types.ts

```diff
@@ -40,7 +40,7 @@ export type GridColumnExtension = {
   /** The name of the column to extend. */
   columnName: string;
   /** The table column width in pixels. */
-  width?: number;
+  width?: number | string;
```

Comment:
Should we add an `auto` type?

Reply:
I don't sure that we should do it. Eventually, we convert all types in string for CSS